src/test/java/seedu/address/ui/PersonCardTest.java

```diff
@@ -36,6 +36,7 @@ public void display() {
             personWithTags.setEmail(ALICE.getEmail());
             personWithTags.setPhone(ALICE.getPhone());
             personWithTags.setTags(ALICE.getTags());
+            personWithTags.setRemark(ALICE.getRemark());
```

Comment:
if this test passes, it seems to me that `setRemark` works fine i.e. the binding works correctly, the `Person` backing the `PersonCard` is updated correctly. That's what `assertCardDisplay` is testing for. However, `assertCardDisplay` doesn't test whether the `remark` will be displayed or not (it only tests that the `Person` backing the `PersonCard` has the correct details). 

As such, try checking the .fxml file and gui components. I think the bug is there. I think only ah, not definitely correct ^^

Reply:
Hey Amos, thanks for your suggestion. I just checked PersonListCard.fxml and it seems to be fine (I think). I just realised another issue though -- apart from the remark not displaying on the GUI, after I've quit the application, the address book.xml does not seem to reflect any change to the remark as well (which was still empty). Could that be one of the affecting bugs?

Reply:
I think you didn't test this properly; I noticed that you didn't update `GuiTestAssert#assertCardDisplaysPerson(ReadOnlyPerson, PersonCardHandle)`. As such you did not verify whether the remarks have been updated.

Reply:
Take a look at line 83 :)

Reply:
Hey Amos, thanks so much for your help -- i've found the bug. I had already updated `GuiTestAssert#asertDisplaysPerson(ReadOnlyPerson, PersonCardHandle)` earlier. However, your suggestion to check if remark has been updated nudged me to modify `PersonCardTest#display`, which for the first time, showed me the remark on the PersonCard. I figured something must be wrong with updating the model since the the remark can actually appear. Eventually I found out that because I did not use `Remark` as a parameter in the constructor, the "updatedPerson" had a blank Remark.